simplified-viewer-epub-readium2/src/main/java/org/nypl/simplified/viewer/epub/readium2/Reader2Bookmarks.kt

```diff
@@ -31,8 +31,8 @@ object Reader2Bookmarks {
   ): ReaderBookmarks {
     return try {
       bookmarkService
-        .bookmarkLoad(accountID, bookID)
-        .get(10L, TimeUnit.SECONDS)
+        .bookmarkSyncAndLoad(accountID, bookID)
+        .get(15L, TimeUnit.SECONDS)
```

Comment:
Why did this value change from 10 to 15?

Reply:
10 seconds seemed to be cutting it close on the emulator, when sometimes things are really slow. In real life it shouldn't matter, but I figured I'd give it some extra time, since it now has to do an additional network request.